cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSISink.java

```diff
@@ -391,38 +391,39 @@ private Status processRollbackedBatches() {
         StringBuffer transactionIds = new StringBuffer();
         batch.startIterator();
         while (batch.hasNext()) {
-            NGSIBatch batchToPersist = new NGSIBatch();
+            //NGSIBatch batchToPersist = new NGSIBatch();
```

Comment:
Lefover comment? Should be removed

Reply:
fixed in https://github.com/telefonicaid/fiware-cygnus/pull/2060/commits/089a8e80613db6eaefdb9a1bc55d8328c142af3f